clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/MigrateLoadBalancerDescription.groovy

```diff
@@ -30,8 +30,6 @@ class MigrateLoadBalancerDescription {
   boolean dryRun
 
   @JsonIgnore
-  NetflixAmazonCredentials getCredentials() {
-    return target.getCredentials();
-  }
+  Set<AccountCredentials> credentialsList = [];
```

Comment:
Yeah, exactly. I will update it so `credentials` is required to be present and can be a collection or an object.